test/middleware.test.js

```diff
@@ -208,6 +208,22 @@ describe('test/middleware.test.js', function () {
       .send('body')
       .expect({}, done);
     });
+
+    it('should parse application/jwt body contents', function (done) {
+      var app = App({
+        enableTypes: ['text', 'json'],
```

Comment:
I think you can use `extendTypes` to parse `application/jwt`, don't need to add jwt to default text type?

```js
var app = App({
  enableTypes: [ 'text', 'json' ],
  extendTypes: {
    text: [ 'application/jwt' ]
  },
});
```

Reply:
Ohh, sorry. I didn't know that this is possible!